### ISSUE TITLE
OrderedDict for python 2.x before 2.7

### ISSUE THREAD

**Issue by medains** [aichallenge/aichallenge#220]
The ants.py code uses OrderedDict which is a Python 2.7 class.

This has been backported to Python 2.4 through 2.6 here:
http://pypi.python.org/pypi/ordereddict

After installing this from pypi I needed to change the following:

``` diff
--- ants/ants.py    2011-07-14 09:54:32.000000000 +0100
+++ ./ants/workingants.py   2011-07-14 10:42:17.000000000 +0100
@@ -1,8 +1,9 @@ 
 #!/usr/bin/env python
 from random import randrange, choice, shuffle, randint, seed
 from math import sqrt
-from collections import deque, defaultdict, OrderedDict
+from collections import deque, defaultdict
 from fractions import Fraction
+from ordereddict import OrderedDict
 import operator
 import string
 from game import Game
```

I'm not really a python programmer, or I'd work out how to sort this out in a more generic and stable fashion

**Comment by xarg** [comment]
Maybe this is better:

try:
    from collections import OrderedDict
except ImportError:
    from ordereddict import OrderedDict

**Comment by medains** [comment]
That does the job very nicely (both the #220 and #221 fixes)

**Comment by jmcarthur** [comment]
Fixed in 66b94de
